Status: published
Date: 2021-12-05 13:49:31
Modified: 2021-12-05 14:00:51
Author: Benjamin Du
Slug: spark-issue:-CalledProcessError:-Command-returned-non-zero-exit-status
Title: Spark Issue: Calledprocesserror: Command Returned Non-Zero Exit Status
Category: Computer Science
Tags: Computer Science, programming, Spark, PySpark, issue, Spark issue, CalledProcessError, non-zero, exit status

**Things on this page are fragmentary and immature notes/thoughts of the author. Please read with your own judgement!**

## Symptom

> subprocess.CalledProcessError: Command './pineapple test --id1-path id1.txt' returned non-zero exit status 1. 

## Cause 

The command invoked by Python failed. 

## Solutions

Figure out why the command invoked by Python failed and fix the issue. 
 
